.changeset/stupid-flowers-tan.md

```diff
@@ -0,0 +1,65 @@
+---
+"graphql-helix": minor
+---
+
+Add better Node.js helpers for building the response for Node.js and fastify.
+
+By using the new response helpers, fastify plugins are fully supported.
+
+The function `toResponsePayload` converts the `processRequest` result into an intermediate format suitable for streaming to clients.
```

Comment:
```suggestion
The function `toResponsePayload` converts the `processRequest` result into an intermediate format suitable for streaming to clients. It contains the HTTP status code, the headers, and an AsyncGenerator that yields the text response that should be streamed to the clients. In a future release this might become the default return type from `processRequest`.
```